huxley/api/tests/test_delegate.py

```diff
@@ -598,3 +598,4 @@ def test_superuser_fail(self):
         ]
 
         self.assertRaises(ValidationError, self.get_response, self.school.id)
+        
```

Comment:
Configure your editor to automatically strip trailing whitespace and add a newline at the end of a file.